src/mame/mame.lst

```diff
@@ -10643,6 +10643,7 @@ mx1600                          // Dynacom MX-1600
 cd6809                          // Codimex CD-6809
 t4426                           // Terco 4426 CNC programming station
 trsvidtx                        // TRS-80 Videotex
+ms1600                          // Instituto Latinoamericano de la Comunicación Educativa / Secretaría de Educación Pública Micro-SEP 1600
```

Comment:
Please keep the lists sorted here.

Reply:
I sorted these to match the ROM definations.

Reply:
No, the lists in this file should be alphabetical.  It’s too hard to keep track of stuff in this “everything bucket” if it’s organised differently per driver group.

Reply:
The `.lst` file should have the entries sorted by shortname – it’s an “everything bucket” and becomes unmanageable otherwise.